integration_test/support/migration.exs

```diff
@@ -14,6 +14,11 @@ defmodule Ecto.Integration.Migration do
       timestamps
     end
 
+    create table(:optimistically_locked_posts) do
```

Comment:
Can you please check if we can use any of the other resources instead of defining a new one? Some are used just in a handful of tests and would be a perfect fit.